src/Disqus.php

```diff
@@ -262,22 +193,16 @@ protected function setOptions(array $options)
         return array_merge($this->options, $options);
     }
 
-    /**
-     * @param string $url
-     * @param mixed  $method
-     *
-     * @return string
-     */
-    protected function httpRequest($url, $method = RequestInterface::METHOD_GET)
+    private function httpRequest(string $url, string $method = 'GET'): ?array
     {
-        $buzz = new Browser(new Curl());
+        $httpClient = HttpClient::create();
 
         try {
-            $response = $buzz->call($url, $method);
-        } catch (\RuntimeException $e) {
-            return false;
+            $response = $httpClient->request($method, $url);
+        } catch (TransportExceptionInterface $e) {
+            return null;
```

Comment:
I kind of think we should just let the `TransportException` be thrown - don't catch it. Users can catch and handle however they want to.